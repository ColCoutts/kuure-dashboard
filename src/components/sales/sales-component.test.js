import React from 'react';
import { shallow } from 'enzyme';
import Sales from './sales-component';

describe('sales component', () => {
  it('renders a snapshot', () => {
    const wrapper = shallow(<Sales />);
    expect(wrapper).toMatchSnapshot();
  });
});
