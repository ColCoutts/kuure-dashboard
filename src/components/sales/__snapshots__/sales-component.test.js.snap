// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sales component renders a snapshot 1`] = `ShallowWrapper {}`;
